### ISSUE TITLE
It seems that versions of electron 22 and higher are not supported.

### ISSUE THREAD

**Issue by INSIinc** [dutchigor/pluggable-electron#16]
It seems that versions of electron 22 and higher are not supported.
```bash
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: flycloudoa@2.0.0
npm ERR! Found: electron@24.4.0
npm ERR! node_modules/electron
npm ERR!   dev electron@"^24.1.3" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer electron@">= 13.x < 22" from pluggable-electron@0.6.0
npm ERR! node_modules/pluggable-electron
npm ERR!   pluggable-electron@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\14031\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\14031\AppData\Local\npm-cache\_logs\2023-05-28T00_50_25_633Z-debug-0.log
```

**Comment by Ytu-str** [comment]
I also encountered this problem, what is the best way? Do you know what conflicts will occur if I upgrade the version

**Comment by dutchigor** [comment]
This is fixed in version 1.0.0.
Tested with Electron version 28 and updated package.json not to invalidate later versions. Please create a new ticket if there are bugs with a later version.
